lib/linter-rust.coffee

```diff
@@ -34,6 +35,11 @@ class LinterRust
           atom.notifications.addError "Invalid specified features",
             detail: "#{err}"
             dismissable: true
+        else
+          if do atom.inDevMode
+            atom.notifications.addError "Something wrong",
```

Comment:
@Arcanemagus is it a proposal for a better error message? If yes, that sounds good, I'll fix it.